### PR TITLE
❇️ Add Polly TTS

### PR DESCRIPTION
**Pull request by rmtuckerphx**
## Proposed Changes

Add Polly TTS

## Types of Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project
- [x] My change requires a change to the documentation
- [x] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

**Comment by rmtuckerphx** [comment]
This includes a change to TtsPlugin as the following audio URL formats are supported by web client:

- `<audio src="https://example.s3.amazonaws.com/polly-tts/audio/hello/en/polly-matthew-06021922f6aa3899aa0954537d955126.mp3"/>`
- `<audio src="SUQzBAAAAAAAI1...VVVVVVV"/>`

But NOT this format:
- `<audio src="data:audio/mpeg;base64,SUQzBAAAAAAAI1...VVVVVVV"/>`

**Comment by aswetlow** [comment]
Oh yeah, we should also support `<audio src="data:audio/mpeg;base64,SUQzBAAAAAAAI1...VVVVVVV"/>` some day. Just `<audio src="SUQzBAAAAAAAI1...VVVVVVV"/>` feels wrong somehow

File: integrations/tts-polly/src/PollyTts.ts [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  DeepPartial,
+  TtsPlugin,
+  Jovo,
+  AudioUtilities,
+  TtsTextType,
+  TtsData,
+} from '@jovotech/framework';
+
+export interface PollyTtsConfig extends TtsPluginConfig {
+  credentials: Credentials;
```
Comment (aswetlow) [inline integrations/tts-polly/src/PollyTts.ts]: Same here: I would make it optional for the Lambda runtime.

**Review by jankoenig — APPROVED** [review]
Thank you @rmtuckerphx!

I changed the docs a bit here and there and also (after some consideration) decided to change `fileExtension` back to `outputFormat` in `TtsPlugin`. This feels more consistent with TTS providers.

In the future, we could think about adding an additional `fileExtension` config to `TtsCache`

**Comment by rmtuckerphx** [comment]
@aswetlow I'm assuming the same comment goes for PollyTts about removing credentials and adding libraryConfig?

**Comment by aswetlow** [comment]
> @aswetlow I'm assuming the same comment goes for PollyTts about removing credentials and adding libraryConfig?

Yes

**Comment by jankoenig** [comment]
Hi @rmtuckerphx,

I updated the integration to work with `libraryConfig`. I'll take a look at #1398 tomorrow.

**Review by aswetlow — APPROVED** [review]

**Pull request merged**